<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="userLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
background-color: rgb(60, 60, 60);
</string>
   </property>
   <property name="placeholderText">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
background-color: rgb(60, 60, 60);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;

background-color: rgb(116, 116, 116);
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="newLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#550000;&quot;&gt;Welcome!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newUserLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
background-color: rgb(60, 60, 60);
</string>
   </property>
   <property name="placeholderText">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
background-color: rgb(60, 60, 60);
</string>
   </property>
   <property name="placeholderText">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
background-color: rgb(60, 60, 60);
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createPB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;

background-color: rgb(116, 116, 116);
</string>
   </property>
   <property name="text">
    <string>Create!</string>
   </property>
  </widget>
  <widget class="QLabel" name="newLabel_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#550000;&quot;&gt;Create New Account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../build-ServicioMensajeria-Desktop_Qt_5_11_3_MinGW_32bit-Debug/kisspng-circle-purple-font-circle-dialog-5aa889c9b910f6.7713059315209947617581.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fondoPrincipal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>401</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>fondoPrincipal.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>fondoPrincipal</zorder>
  <zorder>userLE</zorder>
  <zorder>passwordLE</zorder>
  <zorder>loginPB</zorder>
  <zorder>newLabel</zorder>
  <zorder>newUserLE</zorder>
  <zorder>phoneLE</zorder>
  <zorder>newPasswordLE</zorder>
  <zorder>createPB</zorder>
  <zorder>newLabel_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
